src/yz_solrq.erl

```diff
@@ -70,7 +74,7 @@
     }
 ).
 
-
+%% TODO: Clean this up w/ proper type naming conventions.
```

Comment:
@fadushin or @jvoegele would love to remove this on main branch or w/e works best.